object_cloud/SearchBox search_box
---
object_cloud/DetectedObject[] objects
